### ISSUE TITLE
[product_extended_segmentation] Messages log related issues when cost doesn't get updated

### ISSUE THREAD

**Issue by osvalr** [Vauxoo/addons-vauxoo#799]
### Task [4989](https://www.vauxoo.com/web#id=4989&view_type=form&model=project.task&menu_id=136&action=138)
# Required changes

It is required make a couple of changes:
- When update cost in `standard_price` isn't applied but segments, it's needed to write the log message on `product.product` instead of `product.template`, this happens in [here](https://github.com/Vauxoo/addons-vauxoo/blob/8.0/product_extended_segmentation/model/template.py#L219) as showed below:

**`current code:`**

``` python
...
tmpl_obj.message_post(cr, uid, [product_tmpl_id.id],
                                  'Not Updated Cost, But Segments only.',
                                  'I cowardly did not update Standard new \n'
                                  'price less than old price \n'
...
```
- Include in message log (written in product) the threshold that is based on price delta ( `computed_th` )

**Comment by osvalr** [comment]
@hbto @nhomar can you give me your thoughts about this?

**Comment by hbto** [comment]
Thing is that a template record could be in more than one product record,
thus prompting to update all the Product records linked to the template
is that the idea @nhomar ?

**Comment by nhomar** [comment]
I see the PR he is sending the message in the prodoct.product wher he is working on.

that's the Idea... in what do you need feedback specifically @osvalr

**Comment by osvalr** [comment]
I've discussed with @hbto this because I'm using all bom's related products instead using template's related products, he suggest to take the second choice because some products maybe are not taken in count.

**Comment by nhomar** [comment]
On Mon, Mar 28, 2016 at 4:19 PM, Osval Reyes notifications@github.com
wrote:

> I've discussed with @hbto https://github.com/hbto this because I'm
> using all bom's related products instead using template's related products,
> he suggest to take the second choice because some products maybe are not
> taken in count.

explain more please.

## 

Nhomar Hernandez
CEO Vauxoo.
Site: http://vauxoo.com
Twitter: @nhomar
Blog: http://nhomar.com
Github User: https://github.com/nhomar
Odoo Gold Partner
Skype: nhomar00 (Envia mail previo no lo superviso siempre).
HangOut: nhomar@vauxoo.com
Móvil Venezuela:
+58 4144110269
Móvil México:
+52 1 4773933942

**Comment by osvalr** [comment]
this have been solved in #804
